tools/codegen/dest/lazy_ir.py

```diff
@@ -241,18 +255,32 @@ def this_shape(i: int) -> str:
                 shapes_str = ','.join([this_shape(i) for i in range(returns_length)])
                 meta_out = "std::vector<Shape> shapes{" + shapes_str + "};"
 
-            meta_str = f"""auto out_meta = at::meta::{schema.aten_name}({', '.join(str(a.name) for a in all_args)});
-        {meta_out}"""
+            meta_str = f"""
+            auto out_meta = at::meta::{schema.aten_name}({', '.join(str(a.name) for a in all_args)});
+            {meta_out}"""
         else:
             shape_sig = ComputeShapeSignature(metadata.kernel, func)
             meta_str = f"""
-        auto shapes = {shape_sig.shape_call};"""
+            auto shapes = {shape_sig.shape_call};"""
 
         meta_str += f"""
-        TORCH_INTERNAL_ASSERT(shapes.size() == {returns_length});"""
+            TORCH_INTERNAL_ASSERT(shapes.size() == {returns_length});"""
+
+        symbole_str = aten_symbol(schema)
+        # Rewrite inplace symbol like relu_ into relu
+        # TODO: zero_ is an exception. Need figure out why.
+        if symbole_str == "at::aten::relu_":
+            symbole_str = symbole_str[:-1]
+
+        node_str = f"""torch::lazy::NodePtr node = torch::lazy::ReuseNode<ir::ops::{schema.node_name}>(
+            torch::lazy::OpKind({symbole_str}),
```

Comment:
i was wondering; is it possible inside ReuseNode<T> to call `OpKind(T)` directly using the template arg? (so you can avoid passing in symbol here?)